src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,70 @@
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+import light from "./logo-light-theme.webp";
+import dark from "./logo-dark-theme-removebg-preview-_1_.webp";
+import styles from './header.module.css'
+import styleTog from './toggler.module.css';
+
+function Header({ handleTheme, theme, searchValue, setSearchValue }) {
+  return (
+    <>
```

Comment:
no need for it

Reply:
what do you mean by it ?? do you mean "<>" ?? I delete it, if you mean the objects inside Header I use them in the component.